testprojects/tests/java/org/pantsbuild/testproject/coverage/BUILD

```diff
@@ -0,0 +1,10 @@
+junit_tests(name='all',
```

Comment:
Would recommend against naming this target: better to let the name default to the directory. See https://www.pantsbuild.org/build_files.html#name